go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/alecthomas/jsonschema v0.0.0-20190530235721-fd8d96416671
 	github.com/aws/aws-sdk-go v1.25.14-0.20191218135223-757eeed07291
 	github.com/awslabs/goformation v0.0.0-20190320125420-ac0a17860cf1
```

Comment:
Should we only use v4? and remove this one?

Reply:
This one gets redirected to github.com/errordeveloper/goformation (via a replace directive below), which is Ilya's fork that adds features not present upstream, so we can't replace this with upstream `goformation/v4` without changing the code.